projects/components/src/utils/test/widget-object/angular/angular-widget-finder.ts

```diff
@@ -57,8 +58,8 @@ export class AngularWidgetObjectFinder<H = unknown> {
             new TestElement([ancestor ? ancestor : this.fixture.debugElement], this.fixture)
         );
         let query = widgetConstructor.tagName;
-        if (findOptions?.cssSelector) {
-            query = query + findOptions.cssSelector;
+        if (findOptions) {
+            query = query + SelectorUtil.extractSelector(findOptions);
```

Comment:
This query is passed to `get` as `cssSelector`. So including the data-ui here will make it duplicated, because you are also adding it in a the call to get.

Undo this change, and instead add the data-ui in `parentQuery` below. And please make a similar change in the cypress-widget-finder.

Reply:
Done for both widget-finders.